CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+# [3.5.0](https://github.com/hirosystems/clarinet/compare/v3.4.0...v3.5.0) (2025-08-08)
+
+##### New Features
+
+- use latest explorer (#1954) (e6a624a5)
+- custom boot contracts (#1915) (ae34a3dc)
+
+##### Chores
+
+- make stacks_signer_keys significant for devnet diff (#1950) (017506eb)
+
+##### Other Changes
+
+- add epoch tests (#1945) (9e78c066)
+
+##### Refactors
+
+- rework session api reference (#1951) (b38a5ff2)
```

Comment:
Can you capitilize those bullet point again?
Just to follow the rest of this doc